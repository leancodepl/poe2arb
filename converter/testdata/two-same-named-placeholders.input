[
    {
        "term": "textWithRepeatedNamedPlaceholder",
        "definition": "So {something} is the same thing as {something}.",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    }
]