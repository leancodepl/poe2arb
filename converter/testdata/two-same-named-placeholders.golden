{
    "@@locale": "en",
    "textWithRepeatedNamedPlaceholder": "So {something} is the same thing as {something}.",
    "@textWithRepeatedNamedPlaceholder": {
        "placeholders": {
            "something": {
                "type": "Object"
            }
        }
    }
}
